components/post_markdown/system_message_helpers.jsx

```diff
@@ -46,8 +48,8 @@ function renderLeaveChannelMessage(post, options) {
 }
 
 function renderAddToChannelMessage(post, options) {
-    const username = renderUsername(post.props.username, options);
-    const addedUsername = renderUsername(post.props.addedUsername, options);
+    const username = renderUsernameForUserId(post.user_id, options);
```

Comment:
This will work fine at the moment.
It's not necessary to change on this PR but just like to mention that this will be like below later:
```
const username = renderUsernameForUserId(post.props.userId, options);
```